### DISCUSSION TITLE
Recent Content

### DISCUSSION THREAD

**Discussion by robmoffat** [risk-first/website#88] [Team Posts]
Hi Everyone,

I've not posted in a while on here, but I have been working on a few articles, which you might want to look at.  

The first is in the "estimating" series, https://riskfirst.org/estimating/Risk-First-Analysis  this follows on from https://riskfirst.org/estimating/Fixing-Scrum which has been around a while.

I've also been doing talks recently, and I've put the transcripts up on Risk First.  

cheers,
Rob

**Comment by ghost** [comment]
Hello..! I am Shahbaz Alam from India open source enthusiast,Web Developer, Programmers.
Excited to contribute and learn with all.
